pennylane/fourier/coefficients.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains methods for computing Fourier coefficients and frequency spectra of quantum functions ."""
+from itertools import product
+
+import numpy as np
+
+
+def fourier_coefficients(f, n_inputs, degree, lowpass_filter=False, filter_threshold=None):
+    r"""Computes the first :math:`2d+1` Fourier coefficients of a :math:`2\pi`
+    periodic function, where :math:`d` is the highest desired frequency (the
+    degree) of the Fourier spectrum.
+
```

Comment:
Might be good to copy the latex math equation here. We can't assume readers get here from the module docstring, they might come to this page _first_, and be lacking context.